sdk/template/template/package.json

```diff
@@ -96,34 +93,63 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^4.0.1",
     "@azure/test-utils": "^1.0.0",
-    "@azure-tools/test-credential": "~1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
-    "@microsoft/api-extractor": "^7.31.1",
-    "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-recorder": "^4.0.0",
     "@types/node": "^18.0.0",
-    "chai": "^4.2.0",
     "cross-env": "^7.0.2",
     "dotenv": "^16.0.0",
-    "eslint": "^8.0.0",
-    "esm": "^3.2.18",
     "inherits": "^2.0.3",
-    "karma": "^6.2.0",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-json-preprocessor": "^0.3.3",
-    "karma-json-to-file-reporter": "^1.0.1",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "mocha": "^10.0.0",
-    "nyc": "^15.0.0",
     "rimraf": "^5.0.5",
     "source-map-support": "^0.5.9",
     "typescript": "~5.4.5",
     "util": "^0.12.1",
-    "ts-node": "^10.0.0"
-  }
+    "@vitest/browser": "^1.3.1",
+    "eslint": "^8.56.0",
+    "playwright": "^1.41.2",
+    "prettier": "^3.2.5",
+    "tshy": "^1.11.1",
+    "vitest": "^1.3.1",
+    "loupe": "^3.1.0",
```

Comment:
what is "loupe" used for?

Reply:
Good catch.
playwright or vitest wanted me to install, had to install this to get past one of the blockers.
I'll check again by removing it to make sure it is needed.